### DISCUSSION TITLE
An unexpected internal error occurred

### DISCUSSION THREAD

**Discussion by zainiklab-abdullah** [vercel/community#1537] [Help] [answered]
### Summary

When I am registering in vercel  with my github it show me a mobile verification form.
When I am submitting with valid phone number its showing "An unexpected internal error occurred"

**Comment by swarnava** [comment]
Hi, Can you send us a screenshot?

**Comment by zainiklab-abdullah** [comment]


**Reply by zainiklab-abdullah** [reply]
![image](https://user-images.githubusercontent.com/114804418/219003341-bc34f7cd-88ce-4ed8-ac06-0d3f4314f19d.png)

**Comment by swarnava** [comment, marked as answer]
Can you email at registration@vercel.com using your registered email? Thanks!

**Comment by AbbasAlsbihawi** [comment]
Verification
Please enter your phone number to verify your login. You will receive a single use code to submit.

 
An unexpected internal error occurred
We won’t ever message you — other than for verification purposes.

**Comment by y0u-0** [comment]
same thing here

**Comment by zainiklab-abdullah** [comment]
Hey, my problem was resolved. I have successfully registered.
Thanks.

**Reply by AnasAlMughamsi** [reply]
Could you plesae tell me, how did you solve it ?

**Reply by zainiklab-abdullah** [reply]
Maybe they have fixed that. I just tried and its worked.

**Reply by AbbasAlsbihawi** [reply]
not work ?
